File: src/integration/spring/src/jvmMain/java/community/flock/wirespec/integration/spring/java/configuration/EnableWirespecController.java
package community.flock.wirespec.integration.spring.java.configuration;

import org.springframework.context.annotation.Import;

import java.lang.annotation.*;

@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@Documented
@Import(WirespecSerializationConfiguration.class)
public @interface EnableWirespecController {
}

File: src/integration/spring/src/jvmMain/kotlin/community/flock/wirespec/integration/spring/kotlin/configuration/EnableWirespecController.kt
package community.flock.wirespec.integration.spring.kotlin.configuration

import org.springframework.context.annotation.Import

@Target(AnnotationTarget.CLASS)
@Retention(AnnotationRetention.RUNTIME)
@MustBeDocumented
@Import(WirespecSerializationConfiguration::class)
annotation class EnableWirespecController
